api/routes/groups/groups.ts

```diff
@@ -92,6 +94,52 @@ router.get(
   },
 );
 
+router.patch(
+  '/joinEvent',
+  async (
+    req: Request<
+      {},
+      getResponsesBody<joinEvent>,
+      getRequestBody<joinEvent>,
+      {}
+    >,
+    res: core.Response<getResponsesBody<joinEvent>, {}, getHTTPCode<joinEvent>>,
+  ) => {
+    try {
+      const group = await getRepository(Groups).findOne({
+        where: { id: req.body.groupId },
+        relations: ['events'],
+      });
+
+      console.log(req.body.groupId);
+      console.log(req.body.eventId);
+
+      if (!group) {
+        return res.status(404).json({ msg: 'E_GROUP_DOES_NOT_EXIST' });
+      }
+
+      const newEvent = await getRepository(Event).findOne({
+        id: req.body.eventId,
+      });
+
+      if (!newEvent) {
+        return res.status(404).json({ msg: 'E_EVENT_DOES_NOT_EXIST' });
+      }
+
+      group.events.push(newEvent);
```

Comment:
Maybe check if the group is not already in the event, because maybe the group will appear two times if the event if I call the event two time, it would be interesting to check

Reply:
Actually, it does verify it the group is already in the array.. Wow !